packages/core/src/Table/stories/TableSection/CompleteTableSection.tsx

```diff
@@ -132,7 +133,10 @@ export const CompleteTableSection = () => {
         </HvTable>
       </HvTableContainer>
       {page?.length ? (
-        <HvPagination {...getHvPaginationProps?.()} />
+        <HvPagination
```

Comment:
I think we should add this new pagination to all samples in the table section page